### PR TITLE
[not for merging] concept of a step-wise introduction to the Flutter framework

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

File: _includes/sidebar.html [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
-      <li><a href="/getting-started/">Create and run your first app</a></li>
       <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+    </ul>
+  
+  <li class="sidebar-title">Get started with Flutter</li>
+
+    <ul class="sidebar-items">
+      <li><a href="/getting-started/">Create and run your first app</a></li>
+      <li><a href="/learn/build/">Rendering UI with build()</a></li>
```
Comment (sethladd) [inline _includes/sidebar.html]: Do we mention "build()" here? That's an implementation detail.

I like the idea of the titles that are task oriented. "Rendering UI", "Managing state", etc...

File: getting-started.md [hunk 2]
```diff
@@ -106,14 +106,4 @@ changes are supported by the hot reload feature can be found on the page
...
-
-You might also want to check out:
-* [Building Beautiful UIs with Flutter - Codelab](https://codelabs.developers.google.com/codelabs/flutter/index.html#0)
-* [Introduction to Flutter's Widget Framework](/widgets-intro/)
-
-Happy Fluttering!
-
-
-[mailinglist]: mailto:flutter-dev@googlegroups.com
+Now that you have seen your first running Flutter, let's do a quick journey of core Flutter concepts, starting with [rendering UI with widgets](/learn/build/).
```
Comment (sethladd) [inline getting-started.md]: "run your first Flutter app"

**Comment by sethladd** [comment]
Yes! Love where this is going. Is this something we can land soon? (that is, even if this isn't perfect, it's still a huge step forward)

**Comment by mit-mit** [comment]
I -- obviously :-) -- would love to see this land, but unfortunately don't have the time to complete it. If someone else could take it over, that would be wonderful!

**Comment by mit-mit** [comment]
@mjohnsullivan do we have someone that could take this over and move if forward to landing?

**Comment by mjohnsullivan** [comment]
@mit-mit Yes, I'm happy for one of us to take it over and grow it out. Do we want to revise and commit what yo've done to date and iterate quickly on it, or would you rather we took the branch and worked offline for the moment?

**Comment by mit-mit** [comment]
Fell free to do whatever works best for you!

**Comment by Hixie** [comment]
Can we close this PR in the meantime so it's not on our review queue?

**Comment by mjohnsullivan** [comment]
Sure, as long the branch is still available, we can pull that and the
resubmit a PR once its ready for review/merge.

**Comment by mit-mit** [comment]
OK, closing. The branch is here: https://github.com/flutter/website/tree/mit-mit-gettingstartedconcept

**Pull request closed without merging**